libnavigation-core/src/test/java/com/mapbox/navigation/core/MapboxDistanceFormatterTest.kt

```diff
@@ -0,0 +1,82 @@
+package com.mapbox.navigation.core
+
+import android.content.Context
+import android.graphics.Typeface
+import android.text.style.RelativeSizeSpan
+import android.text.style.StyleSpan
+import androidx.test.core.app.ApplicationProvider
+import com.mapbox.navigation.base.typedef.IMPERIAL
+import com.mapbox.navigation.base.typedef.METRIC
+import com.mapbox.navigation.base.typedef.ROUNDING_INCREMENT_FIFTY
+import org.hamcrest.core.IsInstanceOf
+import org.junit.Assert.assertEquals
+import org.junit.Assert.assertThat
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+import org.robolectric.annotation.Config
+
+@RunWith(RobolectricTestRunner::class)
+@Config(manifest = Config.NONE)
+class MapboxDistanceFormatterTest {
+
+    val ctx = ApplicationProvider.getApplicationContext<Context>()
+
+    @Config(qualifiers = "en")
+    @Test
+    fun formatDistance_when_distance_greaterThan_10_milesOrKilometers() {
```

Comment:
Could we note the relation between the test name and the SUT? We're passing `20000.0` meters to `formatDistance` which is greater than 10 kilometers, could we be more explicit somehow and increase readability / understandability of the test?